src/components/snackbar/Snackbar.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import { useColorMode } from '@chakra-ui/react';
+
+import { closeDark, closeWhite } from '../../assets';
+
+import './snackbar.css';
+
+interface SnackbarProps {
+  type?: 'action' | 'action-long';
+  open: boolean;
+  autoCloseTime?: number;
+  canBeClosed?: boolean;
+  children: React.ReactNode;
+  actionProps?: JSX.Element;
+  align?:
+    | 'top-left'
+    | 'top-right'
+    | 'top-center'
+    | 'bottom-left'
+    | 'bottom-right'
+    | 'bottom-center';
+  onclose: (args: boolean) => void;
+}
+
+function Snackbar({
+  type = undefined,
```

Comment:
Please change or remove undefined as default value.

Reply:
Addressed the comment.